src/Main.java

```diff
@@ -0,0 +1,166 @@
+
+        System.out.println("Метод stringReverse");
+        obj.stringReverse("Hello");
+
+        System.out.println();
+
+        System.out.println("Метод isPalindrome");
+        obj.isPalindrome("ERE");
+        obj.isPalindrome("Allo");
+
+        System.out.println();
+        System.out.println();
+
+        System.out.println("Гра - вгадай слово");
+        obj.guessWord();
+
+
+
+
+
+
+
+
+    }
+
+    public void findSymbolOccurance (String str, char ch) {
+
+        char[] charArray = str.toCharArray();
+        int count = 0;
+
+        for ( int i = 0; i < charArray.length; i++ ) {
+            if ( charArray[i] == ch ) {
+                count++;
+            }
+        }
+
+        System.out.println("Символ зустрічається : " + count + " раз/разів");
+    }
+
+
+    public void findWordPosition (String source, String target) {
+
+        if (source.contains(target)) {
+            System.out.println("Output : " + "1");
+        } else {
+            System.out.println("Output : " + "-1");
+        }
+    }
+
+
+    public void stringReverse (String simpleString) {
+
+        char[] arrayChar = simpleString.toCharArray();
+        String result = "";
+
+        for ( int i = arrayChar.length - 1; i >= 0; i--) {
+            result += String.valueOf(arrayChar[i]);
+        }
+        System.out.println(result);
+
+    }
+
+
+    public void isPalindrome (String checkPalindrome) {
+
+        String lower = checkPalindrome.toLowerCase();
+        char[] tempArray = lower.toCharArray();
+        String reverseWord = "";
+
+        for ( int i = tempArray.length - 1; i >= 0; i--) {
+            reverseWord += String.valueOf(tempArray[i]);
+        }
+
+        if (lower.equals(reverseWord)) {
+            System.out.println("True");
+        } else {
+            System.out.println("False");
+        }
+
+    }
+
+
+    public void guessWord () {
+
+        String[] words = {"apple", "orange", "lemon", "banana", "apricot", "avocado",
+                "broccoli", "carrot", "cherry", "garlic", "grape",
+                "melon", "leak", "kiwi", "mango", "mushroom", "nut",
+                "olive", "pea", "peanut", "pear", "pepper", "pineapple",
+                "pumpkin", "potato"}; // усього 25 слів
+
+
+        Random random = new Random();
+        Scanner input = new Scanner(System.in);
+
+        int randomNum = random.nextInt(25);
+        String word = words[randomNum];
+        String playerChoice = "";
+        // System.out.println(word); перевірка слова
+
+        do {
+            System.out.println("Спробуй вгадати слово, яке я загадав!");
+            playerChoice = input.nextLine().toLowerCase();
+
+            if (playerChoice.equals(word)) {
+                System.out.println("Ти вгадав!");
+                break;
+            } else {
+                System.out.println("Не вгадав! Вот тобі підсказка");
+                sameLetters(word, playerChoice);
+            }
+
+        } while (word != playerChoice) ;
+
```

Comment:
These empty lines should be removed. Please do not forget to format your code before pushing it on a review! This must be done all the time.